service/src/main/java/uk/gov/hmcts/reform/fpl/service/email/content/AdditionalApplicationsUploadedEmailContentProvider.java

```diff
@@ -0,0 +1,121 @@
+package uk.gov.hmcts.reform.fpl.service.email.content;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.fpl.enums.C2AdditionalOrdersRequested;
+import uk.gov.hmcts.reform.fpl.enums.OtherApplicationType;
+import uk.gov.hmcts.reform.fpl.enums.SupplementType;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.common.AdditionalApplicationsBundle;
+import uk.gov.hmcts.reform.fpl.model.common.C2DocumentBundle;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+import uk.gov.hmcts.reform.fpl.model.common.OtherApplicationsBundle;
+import uk.gov.hmcts.reform.fpl.model.notify.BaseCaseNotifyData;
+import uk.gov.hmcts.reform.fpl.model.notify.additionalapplicationsuploaded.AdditionalApplicationsUploadedTemplate;
+import uk.gov.hmcts.reform.fpl.service.email.content.base.AbstractEmailContentProvider;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.apache.commons.lang3.ObjectUtils.isNotEmpty;
+import static uk.gov.hmcts.reform.fpl.enums.TabUrlAnchor.OTHER_APPLICATIONS;
+import static uk.gov.hmcts.reform.fpl.utils.EmailNotificationHelper.buildCallout;
+import static uk.gov.hmcts.reform.fpl.utils.PeopleInCaseHelper.getFirstRespondentLastName;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class AdditionalApplicationsUploadedEmailContentProvider extends AbstractEmailContentProvider {
+
+    public AdditionalApplicationsUploadedTemplate getNotifyData(final CaseData caseData) {
+        return AdditionalApplicationsUploadedTemplate.builder()
+            .callout(buildCallout(caseData))
+            .respondentLastName(getFirstRespondentLastName(caseData.getRespondents1()))
+            .caseUrl(getCaseUrl(caseData.getId(), OTHER_APPLICATIONS))
+            .applicationTypes(getApplicationTypes(caseData.getAdditionalApplicationsBundle().get(0).getValue()))
+            .build();
+    }
+
+    private List<String> getApplicationTypes(AdditionalApplicationsBundle additionalApplicationsBundle) {
+        List<String> applicationTypes = new ArrayList<>();
+        C2DocumentBundle c2DocumentBundle = additionalApplicationsBundle.getC2DocumentBundle();
+        OtherApplicationsBundle otherDocumentBundle = additionalApplicationsBundle.getOtherApplicationsBundle();
+
+        getC2DocumentBundleApplicationTypes(applicationTypes, c2DocumentBundle);
+        getOtherApplicationsBundleApplicationTypes(applicationTypes, otherDocumentBundle);
+
+        return applicationTypes;
+    }
+
+    private void getC2DocumentBundleApplicationTypes(List<String> applicationTypes, C2DocumentBundle c2DocumentBundle) {
```

Comment:
these methods don't return anything, so maybe they should be called addC2DocumentBundleApplicationTypes & addOtherApplicationsBundleApplicationTypes rather than get?